tests/capture/test_capture_mid_measure.py

```diff
@@ -312,8 +312,9 @@ class TestMidMeasureExecute:
     """System-level tests for executing circuits with mid-circuit measurements with program
     capture enabled."""
 
+    @pytest.mark.skip("flaky failures due to single branch statistics")
     @pytest.mark.parametrize("reset", [True, False])
-    @pytest.mark.parametrize("postselect", [None, 0, 1])
+    @pytest.mark.parametrize("postselect", [pytest.param(None, marks=pytest.mark.xfail), 0, 1])
```

Comment:
Could you elaborate on this one?
I suppose it's put in in case the `skip` is removed in the future?
Why does it fail exactly _without_ `postselect`?

Reply:
ended up just xfailing with `strict=False` for this whole thing. Was rather hard to predict which ones always fail and which ones sometimes fail.